e2e-tests/yarn.lock

```diff
@@ -2926,11 +2926,6 @@ to-regex-range@^5.0.1:
   dependencies:
```

Comment:
So this is not a monorepo workspace, right?

Reply:
now it is. Kind of missed that

Reply:
Can't we remove this yarn.lock then?